packages/connectivity/src/scp-cf/destination/destination-accessor.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { pick } from '@sap-cloud-sdk/util';
+import { resolveDestinationWithType } from './destination-accessor';
+import { Destination } from './destination-service-types';
+
+describe('destination accessor', () => {
+  const destination: Destination = {
+    url: 'https://my.system.com',
+    name: 'my-destination',
+    proxyType: 'Internet',
+    username: 'USER_NAME',
+    password: 'password',
+    authentication: 'BasicAuthentication',
+    type: 'HTTP'
+  };
+
+  it('resolves destination when the destination type matches', async () => {
```

Comment:
can you add a type with a mocked mail destination and asserting on the relevant parts for a mail destination.